src/core/Crust/Infobox/Block/HTML/index.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState, useEffect, useRef, useCallback, useLayoutEffect } from "react";
+
+import Icon from "@reearth/components/atoms/Icon";
+import { useT } from "@reearth/i18n";
+import { styled, useTheme } from "@reearth/theme";
+import fonts from "@reearth/theme/fonts";
+
+import { CommonProps as BlockProps } from "..";
+import { Border } from "../utils";
+
+export type Props = BlockProps<Property>;
+
+export type Property = {
+  html?: string;
+  title?: string;
```

Comment:
```suggestion
  default?: {
    html?: string;
    title?: string;
  }
```

Reply:
Why is only htmlblock  wrapped by default property?